app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -1,14 +1,7 @@
-function initNotifications() {
```

Comment:
The linter is complaining about `'use strict';` missing. You just need to add it at the top of the file.